src/CMake/SetUpThirdParty.cmake

```diff
@@ -626,7 +626,12 @@ if(NOT VISIT_BUILD_MINIMAL_PLUGINS OR VISIT_SELECTED_DATABASE_PLUGINS)
 
     include(${VISIT_SOURCE_DIR}/CMake/FindNetcdf.cmake)
 
-    include(${VISIT_SOURCE_DIR}/CMake/FindOpenEXR.cmake)
+    if(VTK_VERSION VERSION_LESS "9.5.0")
+        # linking with vtk 9.5 imposes c++17, and there
```

Comment:
Do we really need this logic? We're at VTK 9.5. after this PR. There is no going back is there?

Reply:
It was how I commented out OpenEXR. It will be changed once the PR for newer openexr is  merged. Yes, I could have done it a different way.